libs/sdk-core/src/lnurl/auth.rs

```diff
@@ -0,0 +1,58 @@
+use std::sync::Arc;
+
+use gl_client::{
+    bitcoin::{
+        hashes::{sha256, Hash, HashEngine, Hmac, HmacEngine},
+        secp256k1::{Message, Secp256k1},
+        util::bip32::ChildNumber,
+    },
+    lightning::util::ser::Writeable,
+};
+use sdk_common::prelude::{LnUrlError, LnUrlResult, LnurAuthSigner};
+
+use crate::node_api::NodeAPI;
+
+pub(crate) struct SDKLnurlAuthSigner {
```

Comment:
Small nit about naming, `Sdk` is used throughout for structs and enums (correct or not, just for consistency)

Reply:
Done